src/events/messageCreate.ts

```diff
@@ -87,7 +87,20 @@ export async function onDirectMessage(
 	}
 	if (tournaments.length === 1) {
 		const tournament = tournaments[0];
-		log("info", msg, { event: "confirm start", tournament: tournament.id });
+		log("info", msg, {
+			event: "confirm start",
+			tournament: tournament.id,
+			size: tournament.players.length,
+			limit: tournament.limit
+		});
+		if (tournament.limit > 0 && tournament.players.length >= tournament.limit) {
+			log("info", msg, { event: "confirm full", tournament: tournament.id });
```

Comment:
Is it worth logging the userId here, and/or alerting private channels so we can get a list of snubs for hosts to contact in the event of a drop? I could see an argument to not, it could be quite spammy.